cmd/scanner/main.go

```diff
@@ -302,6 +302,11 @@ func initServices(ctx context.Context, cfg config.Config) ([]services.Service, e
 	var waitBots int
 	if cfg.LocalModeConfig.Enable {
 		waitBots = len(cfg.LocalModeConfig.BotImages)
+		for _, bot := range cfg.LocalModeConfig.ShardedBots {
```

Comment:
Can we leave a comment above this to explain what it is for? I couldn't make sense of it.

Reply:
The way sharded bots work locally is that there'll be multiple instances of each bot, and scanner dispatches feeds according to their shard id( basically their index). So `waitgroup`s with `len(bots)` will fail because total number of sharded bot containers is actually shards*targets

ps. `shards`*`target`=`num(assignments)` in real world, local mode mimics that behavior